src/main/java/com/iotiq/application/config/CustomerFilter.java

```diff
@@ -0,0 +1,36 @@
+package com.iotiq.application.config;
+
+import com.iotiq.application.domain.enums.Role;
+import com.iotiq.application.service.CustomerService;
+import com.iotiq.user.internal.UserService;
+import jakarta.servlet.Filter;
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.ServletRequest;
+import jakarta.servlet.ServletResponse;
+import lombok.RequiredArgsConstructor;
+import org.springframework.core.Ordered;
+import org.springframework.core.annotation.Order;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+
+@Component
+@Order(Ordered.LOWEST_PRECEDENCE -1)
+@RequiredArgsConstructor
+public class CustomerFilter implements Filter {
```

Comment:
Why are CustomerFilter and SellerFilter different filters? Seems to me one filter can handle both creating a customer and seller.